library/spdm_requester_lib/libspdm_req_get_version.c

```diff
@@ -52,51 +51,43 @@ return_status libspdm_try_get_version(libspdm_context_t *spdm_context,
     libspdm_reset_message_buffer_via_request_code(spdm_context, NULL,
                                                   spdm_request.header.request_response_code);
 
-    status = libspdm_send_spdm_request(spdm_context, NULL,
-                                       sizeof(spdm_request), &spdm_request);
-    if (RETURN_ERROR(status)) {
-        return status;
-    }
+    LIBSPDM_RET_ON_ERR(libspdm_send_spdm_request(spdm_context, NULL,
```

Comment:
I don't have concern to define LIBSPDM_RET_ON_ERR().

But I do have concern to use it in this way, because it makes the step by step debug hard.

I prefer we take 2 steps:
```
    status = libspdm_send_spdm_request(spdm_context, NULL, sizeof(spdm_request), &spdm_request);
    LIBSPDM_RET_ON_ERR(status);
```

As such, when I trace the flow I can know what error status is returned.

Reply:
I concur that this should be split across two statements. It'll make debugging easier.